packages/tools/src/tools/displayTools/Surface/removeSurfaceFromElement.ts

```diff
@@ -1,36 +1,35 @@
 import { getEnabledElement } from '@cornerstonejs/core';
 import type { Types } from '@cornerstonejs/core';
+import { SegmentationRepresentations } from '../../../enums';
 
 /**
  * Remove the surface representation from the viewport's HTML Element.
  * NOTE: This function should not be called directly.
  *
  * @param element - The element that the segmentation is being added to.
- * @param segmentationRepresentationUID - The UID of the surface representation to remove.
- * @param removeFromCache - boolean
+ * @param segmentationId - The id of the segmentation to remove.
  *
  * @internal
  */
-function removeContourFromElement(
+function removeSurfaceFromElement(
   element: HTMLDivElement,
-  segmentationRepresentationUID: string,
-  removeFromCache = false // Todo
+  segmentationId: string
 ): void {
   const enabledElement = getEnabledElement(element);
 
   const { viewport } = enabledElement;
 
   const actorEntries = (viewport as Types.IVolumeViewport).getActors();
 
-  // remove actors whose id has the same prefix as the segmentationRepresentationUID
+  const actorUID = `${segmentationId}-${SegmentationRepresentations.Surface}`;
```

Comment:
Could you please create a utility function which generates the actorUID given the segmentationId and type?  I know it is trivial, but if it is externalized, then it can be updated in the future without changing all the spots you have embedded that knowledge - that is, at the code level you have encapsulated the volatility of hte actorUID name.